### ISSUE TITLE
Rename data providers from `xzyDataProvider` to `providerXyz`

### ISSUE THREAD

**Issue by oliverklee** [MyIntervals/PHP-CSS-Parser#458]

**Comment by JakeQZ** [comment]
Also, while on #456, it seemed to make sense to move all the data providers before the tests:
- Some data providers were used for more than one test, so could not be easily grouped with the relevant test;
- Data providers are `static`, tests are not, also suggested grouping data separately from tests (though within the same file)
  - My personal coding style is to have all `static`s before non-`static`s within a similar classification (still properties before methods - don't know if that's covered by any standard);
- For readability, it seems logical to present the data first:
  - This is the data we're going to use for testing;
  - Now this is what were going to use it for.

WDYT?

**Comment by oliverklee** [comment]
I'm fine with this. I'd prefer to to have the data provider directly above the first test that uses is to reduce scrolling.

**Comment by JakeQZ** [comment]
I just spotted the dxsleyia test in the title.

**Comment by JakeQZ** [comment]
> I'd prefer to to have the data provider directly above the first test that uses is to reduce scrolling.

Agree.  Though In an ideal world where classes are small, that wouldn't be necessary :)
